core/controllers/editor_test.py

```diff
@@ -996,7 +996,8 @@ class ExplorationDeletionRightsTests(BaseEditorControllerTests):
 
     def test_deletion_rights_for_unpublished_exploration(self):
         """Test rights management for deletion of unpublished explorations."""
-        unpublished_exp_id = 'unpub_eid' # Unpublished exploration id.
+        # Unpublished exploration id.
```

Comment:
This and other similar changes have been done because we introduced a lint check which allows trailing (inline) comments only if it's an allowed pragma defined in this [list](https://github.com/oppia/oppia/blob/fe9ab48ef897c83ae3b3b593399bc3ebb4699fe5/scripts/linters/pylint_extensions.py#L49).